server/fty-srr.cfg

```diff
@@ -13,5 +13,5 @@ srr-msg-bus
 
 
 srr
-    version = 2.1 # Srr version.
+    version = 2.2 # Srr version.
```

Comment:
Though correct, there is probably no need for advancing version, since the new features are both in separate groups and do not alter the save/restore sequence inside a group.
- when restoring new payload on IPM < 2.4.0: the new features will be ignored (as the check is done over the hardcoded sequence of known groups)
- when restoring older payload on > 2.4.0: the groups will be missing and will simply not be restored (normal behavior for backward compatibility)

Reply:
@perrettecl, I think we adopted the same approach for the virtualization settings, am I right?